core/api/src/app/authentication/login.ts

```diff
@@ -400,3 +401,36 @@ const checkDeviceLoginAttemptPerAppcheckJtiLimits = async (
     rateLimitConfig: RateLimitConfig.deviceAccountCreate,
     keyToConsume: appcheckJti,
   })
+
+export const getAuthTokenFromUserId = async (
+  userId: UserId,
+): Promise<AuthToken | AuthenticationError> => {
+  const { data } = await kratosAdmin.getIdentity({ id: userId })
+  let authToken = null
+  let kratosResult = null
```

Comment:
type is missing. 

@vindard would be interesting to know why Typescript doesn't complain